src/Context.cpp

```diff
@@ -1195,6 +1123,7 @@ void Context::createDefaultConfig() {
              << "#include solarized-dark-256.theme\n"
              << "#include solarized-light-256.theme\n"
              << "#include no-color.theme\n"
+             << "#include default.theme\n"
```

Comment:
This should be un-commented:
```suggestion
             << "include default.theme\n"
```